storage/database/db_manager.go

```diff
@@ -111,6 +116,14 @@ type DBManager interface {
 	HasStateTrieNode(key []byte) (bool, error)
 	ReadPreimage(hash common.Hash) []byte
 
+	// Read StateTrie from new DB
+	ReadCachedTrieNodeFromNew(hash common.Hash) ([]byte, error)
+	ReadCachedTrieNodePreimageFromNew(secureKey []byte) ([]byte, error)
+	ReadStateTrieNodeFromNew(key []byte) ([]byte, error)
+	HasStateTrieNodeFromNew(key []byte) (bool, error)
+	ReadPreimageFromNew(hash common.Hash) []byte
```

Comment:
Can you put a more precise adjective instead of `new`? Every new thing finally gets old.

Reply:
I can't find some good adjective. Please recommend good adjective. :)
I will change the adjective word in other PR.

Reply:
As we talked, I will reflect like below. :)
```
ReadCachedTrieNodeFromNew -> ReadCachedTrieNodeFromMigrationDB
```